src/Component.php

```diff
@@ -2,15 +2,40 @@
 
 declare(strict_types=1);
 
-namespace MyComponent;
+namespace Keboola\TelemetryData;
 
 use Keboola\Component\BaseComponent;
+use Keboola\Component\Logger;
+use Keboola\Component\Manifest\ManifestManager;
 
 class Component extends BaseComponent
 {
     protected function run(): void
     {
-        // @TODO implement
+        $extractor = new Extractor(
+            new DbConnector($this->getAppConfig(), $this->getAppLogger()),
```

Comment:
`$this->getAppLogger()` -> `$this->getLogger()` ... mal by si pouzivat `LoggerInterface` namiesto `Logger` ... mozes tak potom pouzit lubovolny logger, ... hodi sa to napr. pri testoch ... a budes mat o jednu zbytocnu metodu menej.